src/components/Nav/Navigation.tsx

```diff
@@ -56,7 +56,12 @@ export class Navigation extends React.Component<PropsType, NavigationState> {
   }
 
   componentDidMount() {
-    document.title = serverConfig.installationTag ? serverConfig.installationTag : 'Kiali Console';
+    let pageTitle = serverConfig.installationTag ? serverConfig.installationTag : 'Kiali';
```

Comment:
I see the installation tag is still here and not being overridden but the `[cluster id]` is appended to it. So this is good.